governance/roles.md

```diff
@@ -283,4 +289,37 @@ Some ongoing projects may have teams where members are identified for
 additional roles and may be required to have specific expertise. For visibility,
 these additional project roles are listed below:
 
-* [Security Reviewers](../assessments/guide/security-reviewer.md/)
+## Lead rotations
+
+For certain facilitator roles, teams and projects that are ongoing, leadership
+rotations should be carried out every 6 months. This is to:
+
+* Ensure leadership continuity of projects and passing on of tribal knowledge
+* Ensure diverse perspective/breadth of knowledge
+* Prevention of commitment bias in projects
+* Provide more leadership opportunities for more community members
+
+A facilitator role/project lead position should be a candidate for rotation if the
+project/facilitation task:
+
+* Exists for more than 6 months
+* Involves more than 1 person
+
```

Comment:
Might want to clarify here a little - is rotation mandatory? What if current lead is happily doing their thing and Good Work is being accomplished?

Reply:
I think a maximum term length is healthy. Else it will become like a benevolent dictatorship and fails to bring in diversity and shared ownership.

Reply:
@jlk The rotations are done in good faith. In such a case, one of the things we've done in the past for Cloud Native Security day is to add more co-leads in (which worked for the project as it was expanding as well). That is a good note and should be spelled out as a possible outcome. 

That still has a possibility of creating groupthink as @mkbhanda has pointed out, but its a tricky balancing act as with any open source project. The onus is still on the co-chairs to evaluating the situation.